content/_data/changelogs/weekly.yml

```diff
@@ -6293,6 +6293,189 @@
   # pull: 4494 (PR title: Remove references to Azure maven cache)
   # pull: 4502 (PR title: [JENKINS-61102] Use XMLUnit to fix test)
 
+- version: '2.222'
+  date: 2020-02-23
+  changes:
+  - type: major rfe
+    category: major rfe
+    pull: 4463
+    issue: 60920
+    authors:
+    - fqueiruga
+    - daniel-beck
+    message: |-
+      Revamp the layout and icons of the header bar and breadcrumbs.
+      Instances with plugins that depend on details of the Jenkins layout (e.g. Simple Theme Plugin) may experience UI/layout problems.
+      A new experimental header color scheme can be enabled by setting the <code>jenkins.ui.refresh</code> system property to <code>true</code>.
+  - type: major rfe
+    category: major rfe
+    pull: 4463
+    authors:
+    - fqueiruga
+    - daniel-beck
+    message: |-
+      Introduce a new experimental UI that can be enabled by setting the <code>jenkins.ui.refresh</code> system property to <code>true</code>.
+      Currently it includes a new header color scheme, more changes to be added as a part of the UI/UX revamp.
+    references:
+      - pull: 4463
+      - issue: 60920
+      - url: https://github.com/jenkinsci/jep/blob/master/jep/223/README.adoc
+        title: JEP-223
+      - url: https://jenkins.io/sigs/ux/
+        title: Jenkins UX SIG
+  - type: major rfe
+    category: major rfe
+    pull: 4501
+    authors:
+    - daniel-beck
+    - timja
+    references:
+      - pull: 4501
+      - issue: 60266
+      - url: https://github.com/jenkinsci/jep/blob/master/jep/223/README.adoc
+        title: JEP-223
+    message: |-
+      Add a new experimental <code>Overall/Manage</code> permission which allows a user to configure parts of the global Jenkins configuration without having the <code>Overall/Administer</code> permission.
+      This is an experimental feature, disabled by default, that can be enabled by setting the <code>jenkins.security.ManagePermission</code> system property to <code>true</code>.
```

Comment:
Do we need to add that it's Beta restricted also ? Or is it enough to say experimental for the release note ?

Reply:
I think so. `Beta` is for developers, it has no user frontend

Reply:
yes, we will include that `@Beta` in our developer docs